net/crowdsec-firewall-bouncer/files/crowdsec-firewall-bouncer.initd

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh /etc/rc.common
+# (C) 2021 Gerald Kerma
+
+START=99
+USE_PROCD=1
+NAME=crowdsec-firewall-bouncer
+PROG=/usr/bin/cs-firewall-bouncer
+CONFIG=/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+BACKEND=iptables
+VARCONFIGDIR=/var/etc/crowdsec/bouncers
+VARCONFIG=/var/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+FW_BACKEND="iptables"
+
+service_triggers() {
+	procd_add_reload_trigger crowdsec-firewall-bouncer
+}
+
+init_config() {
+	config_load crowdsec
+	config_get BACKEND crowdsec backend "${BACKEND}"
+
+	if [ "$BACKEND" = "auto" ]; then 
+		## CheckFirewall
+		iptables="true"
+		which iptables > /dev/null
+		FW_BACKEND=""
+		if [[ $? != 0 ]]; then
+		  echo "iptables is not present"
+		  iptables="false"
+		else
+		  FW_BACKEND="iptables"
+		  echo "iptables found"
+		fi
+
+		nftables="true"
+		which nft > /dev/null
+		if [[ $? != 0 ]]; then
+		  echo "nftables is not present"
+		  nftables="false"
+		else
+		  FW_BACKEND="nftables"
+		  echo "nftables found"
+		fi
+
+		if [ "$nftables" = "true" -a "$iptables" = "true" ]; then
+		  echo "Found nftables(default) and iptables..."
+		fi
+
+		if [ "$FW_BACKEND" = "iptables" ]; then
+		  which ipset > /dev/null
+		  if [[ $? != 0 ]]; then
+		    echo "ipset not found, install it !"
+		  fi
+		fi
```

Comment:
This check for ipset right now is only done if the backend value from uci is `"auto"`. I think it should also be done if the user manually set the uci value to `"iptables"`, i.e. outside of/after the current `if` block.

Reply:
not sure to understand...
The if block test is for iptables, then, inside it, we test ipset ...

Reply:
okay, understood, it was in the case of manual select by user...
It is now removed and only auto detected... so must be ok !
right ?
Thanks...